src/mongo/client/dbclientinterface.h

```diff
@@ -1145,7 +1145,8 @@ namespace mongo {
          *  }
          */
         std::auto_ptr<DBClientCursor> enumerateCollections( const std::string& db,
-                                                            const BSONObj& filter = BSONObj() );
+                                                            const BSONObj& filter = BSONObj(),
+                                                            int batchSize = 0 );
```

Comment:
Kind of a shame to have this wart in the interface just for testing. No user should ever care. However, I can live with it I guess.